src/harness/testcases/WINNF_FT_S_EXZ_testcase.py

```diff
@@ -293,4 +285,258 @@ def test_WINNF_FT_S_EXZ_1(self,config_filename):
+    device_N1_1['installationParam']['longitude'] = -96.25100
+
+    device_N1_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    # Moving device_N1_2 just outside of 50 meters from exclusion zone of
+    # 'West Combined Contour'
+    device_N1_2['installationParam']['latitude'] = 33.58101 
+    device_N1_2['installationParam']['longitude'] = -114.35775 
+
+    device_N1_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    # Moving device_N1_3 just outside of 50 meters from exclusion zone of
+    # 'West Combined Contour'
+    device_N1_3['installationParam']['latitude'] = 33.85263
+    device_N1_3['installationParam']['longitude'] = -106.57198
+
+    # Loading N2 set of CBSDs each located within at least one Exclusion Zone
+    device_N2_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    # Moving device_N2_1 to a location inside the exclusion zone of
+    # 'West Combined Contour'
+    device_N2_1['installationParam']['latitude'] = 37.26531 
+    device_N2_1['installationParam']['longitude'] = -121.89331
+    device_N2_1['fccId'] = "test_fcc_id_bb"
+    device_N2_1['cbsdSerialNumber'] = "test_serial_number_bb"
+
+    device_N2_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    # Moving device_N2_2 to a location inside the exclusion zone of
+    # 'East-Gulf Combined Contour'
+    device_N2_2['installationParam']['latitude'] = 30.69461
+    device_N2_2['installationParam']['longitude'] = -89.82421 
+    device_N2_2['fccId'] = "test_fcc_id_cc"
+    device_N2_2['cbsdSerialNumber'] = "test_serial_number_cc"
+
+    # Loading N3 set of CBSDs each located within 50 meters of at least one Exclusion Zone
+    device_N3_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    # Moving device_N3_1 to a location within 50 meters from the exclusion zone
+    # of 'East-Gulf Combined Contour'
+    device_N3_1['installationParam']['latitude'] = 43.94141
+    device_N3_1['installationParam']['longitude'] = -71.50314
+    device_N3_1['fccId'] = "test_fcc_id_aaa"
+    device_N3_1['cbsdSerialNumber'] = "test_serial_number_aaa"
+
+    device_N3_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    # Moving device_N3_2 to a location within 50 meters from the exclusion zone
+    # of 'West Combined Contour'
+    device_N3_2['installationParam']['latitude'] = 47.35184
+    device_N3_2['installationParam']['longitude'] = -123.68957
+    device_N3_2['fccId'] = "test_fcc_id_bbb"
+    device_N3_2['cbsdSerialNumber'] = "test_serial_number_bbb"
+
+    device_N3_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    # Moving device_N3_3 to a location within 50 meters from the exclusion zone
+    # of 'East-Gulf Combined Contour'
+    device_N3_3['installationParam']['latitude'] = 37.78916
```

Comment:
device_N3_3, device_N3_2, device_N3_1 are all within 50 "kilometers" and not 50 "meters". Could you pls double check these 3 device locations. Thanks.

Reply:
Thanks, fixed.